### ISSUE TITLE
Login window error

### ISSUE THREAD

**Issue by Scarlov87** [sketchfab/unreal-plugin#33]
Hello Guys I'm trying to log in Sketchfab but it drops the next message

"Forbidden (403)
CSRF verification failed. Request aborted.
You are seeing this message because this site requires a CSRF cookie when submitting forms. This cookie is required for security reasons, to ensure that your browser is not being hijacked by third parties.
If you have configured your browser to disable cookies, please re-enable them, at least for this site, or for “same-origin” requests.
More information is available with DEBUG=True." 
![image](https://user-images.githubusercontent.com/106821782/177660737-58152fd0-fc90-4e52-8dba-af52c6c27a65.png)
I'm a noob here so I don't know how to enable cookies on this one :) thanks a lot for your help :)

**Comment by matiassr4** [comment]
wait for sketchfab upgrade, this error is because the plugin is out to date.

**Comment by MrKylePopovich** [comment]
I'm having this issue too, two days now. 
Is it out of date or some sort of other issue? Server?

**Comment by Scarlov87** [comment]
But there must be a way to enable cookies right? is it linked to any local browser? I mean like chrome or something?

**Comment by MrKylePopovich** [comment]
> 

yes, it's linked to whatever browser is set to your computers default (in my case edge),
possible solution!? thanks for hoping on this, Scarlov

**Comment by josegoval** [comment]
Any news about this?

**Comment by Sunloggy** [comment]
Still stuck at login, I change my default web browser, nothing change:((

**Comment by Lychenkov** [comment]
Same error

**Comment by slamjs** [comment]
I see just the infinite loading

**Comment by ZacDair** [comment]
Same issue as - https://github.com/sketchfab/unreal-plugin/issues/5

My original workaround was to manually import models (it's not by any means, but at least I could progress my project)
